t/089-phase.t

```diff
@@ -8,7 +8,7 @@ log_level('warn');
 
 repeat_each(2);
 
-plan tests => repeat_each() * (blocks() * 2 + 1);
+plan tests => repeat_each() * (blocks() * 2) + 3;
```

Comment:
This is a bad test plan because it won't respect changes in `repeat_each()`. Please update it, and ideally, modify your new test so that it has the same number of tests as other ones in this suite.